studio/schemas/documents/aboutSection.js

```diff
@@ -0,0 +1,25 @@
+export default {
+    name: 'aboutSection',
+    title: 'About Section',
+    type: 'document',
+    fields: [
+        {
+            name: 'leftAboutText',
+            title: 'Left About Text',
+            type: 'array',
+            of: [{ type: 'block' }],
+        },
```

Comment:
should probably be `leftColumn`/`rightColumn`. I think text is implied so I don't think it's necessary in the name.